metadata.json

```diff
@@ -48,6 +48,9 @@
     },
     {
       "operatingsystem": "Archlinux"
+    },
+    {
+      "operatingsystem": "FreeBSD"
```

Comment:
can you list the freebsd versions that this module works on? After that, our unit tests will automatically test against those versions.